frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -0,0 +1,17 @@
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  className?: 'yellow' | 'black';
+  children: React.ReactNode;
+};
+
+const PageLayout: React.FC<Properties> = ({
+  children,
+  className = 'yellow',
+}: Properties) => (
+  <div className={clsx(styles.pageLayout, styles[className])}>{children}</div>
```

Comment:
I meant something like that (adding default yellow background to pageLayout, and rewriting it by custom class in place where it's needed)
```suggestion
  className?: string;
  children: React.ReactNode;
};

const PageLayout: React.FC<Properties> = ({
  children,
  className,
}: Properties) => (
  <div className={clsx(styles.pageLayout, className)}>{children}</div>
```

Reply:
Done